### PR TITLE
Change maintainer from @brian-brazil to @fstab and @tomwilkie

### PR DESCRIPTION
**Pull request by beorn7**
Yesterday at the dev-summit, the Prometheus team has proposed @fstab and @tomwilkie as the new maintainer for this repo after @brian-brazil's retirement from the project.

This change will only be merged after an announcement on the prometheus-developers mailing list and is subject to lazy consensus.

Thank you, @brian-brazil, for the incredible amount of work done in this repository and for the project in general.

### CODE REVIEW

**Comment by beorn7** [comment]
@tomwilkie please verify that I have used your preferred email address for this purpose.

@fstab please verify that I have used your preferred email address for this purpose. And please approve, if you agree with your nomination. (I cannot mark you as a reviewer yet, but this will change once this is merged and we add you to the org.)

**Review by tomwilkie — APPROVED** [review]

**Comment by beorn7** [comment]
@fstab friendly reminder to review this and, once you are happy, to approve.

File: MAINTAINERS.md
Comment (fstab) [inline MAINTAINERS.md:1]: Please use fabian@fstab.de

Comment (beorn7) [reply MAINTAINERS.md]: Done.

**Review by fstab — APPROVED** [review]

**Pull request merged**
